jupyter_server/base/zmqhandlers.py

```diff
@@ -239,6 +239,11 @@ def _reserialize_reply(self, msg_or_list, channel=None):
             smsg = json.dumps(msg, default=json_default)
             return cast_unicode(smsg)
 
+    def select_subprotocol(self, subprotocols):
+        selected_subprotocol = "0.0.1" if "0.0.1" in subprotocols else None
```

Comment:
According to the [guidance](https://datatracker.ietf.org/doc/html/rfc6455#section-1.9) (and the examples in the [registry](https://www.iana.org/assignments/websocket/websocket.xhtml#subprotocol-name)), we should probably name this subprotocol something like `v1.websocket.jupyter.org`.